include/aos/common/cryptoutils.hpp

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CRYPTOUTILS_HPP_
+#define AOS_CRYPTOUTILS_HPP_
+
+#include "aos/common/crypto.hpp"
+#include "aos/common/pkcs11/pkcs11.hpp"
+#include "aos/common/uuid.hpp"
+
+namespace aos {
+namespace cryptoutils {
+
+/**
+ * Loads certificates and keys by URL
+ */
+class CertLoader {
+public:
+    /**
+     * Constructs object instance.
+     *
+     * @param cryptoProvider crypto provider interface.
+     * @param pkcs11Context PKCS11 session context.
+     * @param uuidManager uuid manager.
+     * @param allocator allocator.
+     */
+    CertLoader(crypto::x509::ProviderItf& cryptoProvider, pkcs11::SessionContext& pkcs11Context,
```

Comment:
It should not take pkcs11Context and allocator. All, interfaces should be passed through `Init` function.

Reply:
Fixed